README.md

```diff
@@ -7,4 +7,13 @@ git push origin master
 
 git pull origin master
 ```
-llklskds
\ No newline at end of file
+## trabajo con ramas
+```sh
+git branch nombre_rama # crear rama
+git checkout nombre_rama # pasarse a la rama
+git branch # listar las ramas
+
+git add . 
+git commit  -m "el mensaje"
+git push origin nombre_rama
+```
```

Comment:
please add the new command `git commit --amend`

Reply:
ok agregado.